src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AbstractDriverAdapter.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.AbstractDriver;
+
+public class AbstractDriverAdapter extends AbstractDriver {
+
+    private int startX = 0, startY = 0;
+    private DrawPanelController drawPanelController;
+
+    public AbstractDriverAdapter(DrawPanelController drawPanelController) {
+        super(0, 0);
+        this.drawPanelController = drawPanelController;
+    }
+
+    @Override public void operateTo(int x, int y) {
+        ILine line = LineFactory.getDottedLine();
+        line.setStartCoordinates(startX, startY);
```

Comment:
Does not work correctly. startX and startY are not updated by SetPosition calls (superclass). GetX and GetY should be used.